### ISSUE TITLE
Arrange device animation

### ISSUE THREAD

**Issue by aptonline** [tuanha2000vn/hasskit#60]
When editing devices the animation shown is very erratic and causes discomfort to the eyes.  Would it be possible to slow down the animation and make more like the Apple home app animation (see below).

![2BFF8849-1428-4C72-8183-04AD7A9CBDB1](https://user-images.githubusercontent.com/190136/71721406-ce1a1800-2e1c-11ea-8674-c1df0fa3b5c5.gif)

**Comment by tuanha2000vn** [comment]
Yeah, the code for wiggle icon I found somewhere in the net and it look scary.

I'll try to find better code and replace if there's any.

**Comment by aptonline** [comment]
Not sure if these will be useful.

https://inducesmile.com/google-flutter/how-to-create-a-shake-animation-in-flutter/
https://www.didierboelens.com/2018/06/animations-in-flutter---easy-guide---tutorial/

**Comment by tuanha2000vn** [comment]
Till no good premade wiggle animation...

**Comment by aptonline** [comment]
No, you’d have to get creative and use the techniques above 😉
